SEED-platform/seed
# Add property timezone as a database field

#### Any background context you want to provide?
Meters need to know the timezone in order to have the correct time stamps in the meter data. It made sense to add the timezone string to the property which makes the assumption that the meters are all within the property and that the property does not span multiple time zones.

#### What's this PR do?
* Add property_timezone
* Update `example-data-properties.xlsx` with a column for timezones
* Format of the timezone needs to follow the `pytz.all_timezones` format. 

#### How should this be manually tested?
* import new data that has a column for timezone
* map to new field
*Note that the timezone string is not validated. Downstream code will need to ensure the string is in correct format*

#### What are the relevant tickets?
#3243 

#### Screenshots (if appropriate)
 
![image](https://user-images.githubusercontent.com/1907354/170737366-8f208546-cbb5-4e1b-8c1b-7fd636343a42.png)
